### PR TITLE
WIP: Chocolate

### PR DESCRIPTION
**Pull request by pavelsavara**

### CODE REVIEW

**Comment by flemming-n-larsen** [comment]
@pavelsavara Your addition for Chocolate looks really good. 👍 
I think we can merge this one if you think it is ready?
Beware that 1.9.4.2 is already out, so we can roll this to version 1.9.4.3.

**Comment by pavelsavara** [comment]
We need to become owners of https://community.chocolatey.org/packages/robocode first.
I already contacted the current owner in April, but no response.
So now I reached to Chocho admins.

**Comment by flemming-n-larsen** [comment]
@pavelsavara Cool. No rush. Lets us see if we are lucky with the Choco admins. 👍

**Comment by pavelsavara** [comment]
@flemming-n-larsen , what's your chocolatey account ? I would like to add you as admin :)
https://community.chocolatey.org/packages/robocode

**Comment by flemming-n-larsen** [comment]
@pavelsavara I just created an account: fnl
You should be able to use my email address as well. :)

**Comment by pavelsavara** [comment]
You are added. You will receive email about accepting it. I missed mine in spam folder.

**Comment by flemming-n-larsen** [comment]
Thanks. I am now added as admin :)

File: robocode.installer/src/chocolatey/robocode.nuspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>robocode</id>
+    <version>1.1.1.1</version>
```
Comment (pavelsavara) [inline robocode.installer/src/chocolatey/robocode.nuspec:5]: I don't remember if this is somehow auto-modified or needs attention

**Comment by pavelsavara** [comment]
Feel free to take over Flemming

File: robocode.installer/src/chocolatey/robocode.nuspec (hunk 1 above)
Comment (flemming-n-larsen) [reply robocode.installer/src/chocolatey/robocode.nuspec:5]: @pavelsavara Thank you for paying attention to this detail. :)
I have just verified the `gradlew chocoBuild` and checked that the version of `robocode.nuspec` inside the .nupkg file gets the correct version. So currently 1.1.1.1 is being replaced with 1.9.4.2.

**Review by flemming-n-larsen — APPROVED** [review]

**Pull request merged**